package.json

```diff
@@ -1,29 +1,43 @@
 {
-  "name": "gt-scheduler",
+  "name": "gt-scheduler-v2",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://jasonpark.me/gt-scheduler",
+  "homepage": "https://gtscheduler.tech/",
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.7",
-    "@fortawesome/free-solid-svg-icons": "^5.4.2",
-    "@fortawesome/react-fontawesome": "^0.1.3",
-    "axios": "^0.18.0",
+    "@emotion/core": "^10.0.28",
+    "@emotion/styled": "^10.0.27",
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
+    "@fortawesome/free-solid-svg-icons": "^5.13.0",
+    "@fortawesome/react-fontawesome": "^0.1.10",
+    "@progress/kendo-react-popup": "^3.14.0",
+    "ajv": "^6.12.2",
+    "axios": "^0.18.1",
+    "cheerio": "^1.0.0-rc.3",
+    "cors-anywhere": "^0.4.3",
     "dom-to-image": "^2.6.0",
     "fast-html-parser": "^1.0.1",
-    "file-saver": "^2.0.0-rc.4",
-    "github-fork-ribbon-css": "^0.2.1",
-    "js-cookie": "^2.2.0",
-    "memoize-one": "^4.0.2",
-    "node-sass": "^4.9.4",
+    "file-saver": "^2.0.2",
+    "github-fork-ribbon-css": "^0.2.3",
+    "http": "0.0.1-security",
+    "jquery": "^3.5.1",
+    "js-cookie": "^2.2.1",
+    "memoize-one": "^4.0.3",
+    "node-sass": "^4.14.1",
     "normalize.css": "^8.0.1",
-    "react": "^16.5.2",
-    "react-copy-to-clipboard": "^5.0.1",
-    "react-dom": "^16.5.2",
-    "react-redux": "^5.1.0",
-    "react-scripts": "2.0.5",
-    "react-virtualized": "^9.21.0",
-    "redux": "^4.0.1",
-    "redux-actions": "^2.6.3"
+    "react": "^16.13.1",
+    "react-copy-to-clipboard": "^5.0.2",
+    "react-dom": "^16.13.1",
+    "react-redux": "^5.1.2",
+    "react-resize-panel": "^0.3.5",
+    "react-scripts": "^3.4.1",
+    "react-virtualized": "^9.21.2",
+    "redux": "^4.0.5",
+    "redux-actions": "^2.6.5",
+    "request": "^2.88.2",
+    "rmp-api": "0.0.1-security",
+    "semantic-ui-react": "^0.88.2",
+    "typescript": "^3.9.3",
+    "underscore": "^1.10.2"
```

Comment:
I see a lot of packages that are potentially not in use:
- `@emotion/core`
- `@emotion/styled`
- `@progress/kendo-react-popup`
- `ajv`
- `cors-anywhere`
- `request`
- `rmp-api`
- `semantic-ui-react`
- `typescript`
- `underscore`

They should be removed.